tests/func/test_state.py

```diff
@@ -69,16 +70,33 @@ def test_get_state_record_for_inode(get_inode_mock, tmp_dir, dvc):
         assert ret is not None
 
 
-def test_remove_unused_links(tmp_dir, dvc):
-    assert len(tmp_dir.dvc_gen("foo", "foo_content")) == 1
-    assert len(tmp_dir.dvc_gen("bar", "bar_content")) == 1
+def test_remove_links(tmp_dir, dvc):
+    tmp_dir.dvc_gen({"foo": "foo_content", "bar": "bar_content"})
 
-    cmd_count_links = "SELECT count(*) FROM {}".format(State.LINK_STATE_TABLE)
     with dvc.state:
+        cmd_count_links = "SELECT count(*) FROM {}".format(
+            State.LINK_STATE_TABLE
+        )
         result = dvc.state._execute(cmd_count_links).fetchone()[0]
         assert result == 2
 
-        dvc.state.remove_unused_links([])
+        dvc.state.remove_links(["foo", "bar"])
 
         result = dvc.state._execute(cmd_count_links).fetchone()[0]
         assert result == 0
+
+
+def test_get_unused_links(tmp_dir, dvc):
+    tmp_dir.dvc_gen({"foo": "foo_content", "bar": "bar_content"})
+
+    with dvc.state:
+        links = [os.path.join(dvc.root_dir, link) for link in ["foo", "bar"]]
```

Comment:
Why using absolute paths?

Reply:
`get_unused_links` already expect it to be a abspath. So, I'd not like to change it here (at least, not in this PR).